github/github.go

```diff
@@ -24,14 +26,14 @@ func NewClient() *Client {
 	}
 }
 
-func (c *Client) SetToken(token string) {
-	c.client = c.client.WithAuthToken(token)
+func (c *Client) RefreshClientWithToken(token string) {
+	c.client = github.NewClient(nil).WithAuthToken(token)
```

Comment:
Fixing bug from previous PR, when setting a token, a new client must be initialized.